com.unity.cinemachine/Editor/Windows/CinemachineSettings.cs

```diff
@@ -6,293 +6,180 @@ namespace Cinemachine.Editor
-                    return UnpackColour(packedColour);
-                }
+                byte[] bytes = new byte[16];
+                byte[] rBytes = BitConverter.GetBytes(col.r);
+                byte[] gBytes = BitConverter.GetBytes(col.g);
+                byte[] bBytes = BitConverter.GetBytes(col.b);
+                byte[] aBytes = BitConverter.GetBytes(col.a);
 
-                set
-                {
-                    if (SoftBoundsOverlayColour != value)
-                    {
-                        string packedColour = PackColor(value);
-                        EditorPrefs.SetString(kComposerSoftBoundsColourKey, packedColour);
-                    }
-                }
-            }
-
-            public static Color TargetColour
-            {
-                get
-                {
-                    string packedColour = EditorPrefs.GetString(kComposerTargetColourKey, PackColor(kDefaultTargetColour));
-                    return UnpackColour(packedColour);
-                }
+                Buffer.BlockCopy(rBytes, 0, bytes, 0, 4);
+                Buffer.BlockCopy(gBytes, 0, bytes, 4, 4);
+                Buffer.BlockCopy(bBytes, 0, bytes, 8, 4);
+                Buffer.BlockCopy(aBytes, 0, bytes, 12, 4);
 
-                set
-                {
-                    if (TargetColour != value)
-                    {
-                        string packedColour = PackColor(value);
-                        EditorPrefs.SetString(kComposerTargetColourKey, packedColour);
-                    }
-                }
+                return Base64Encode(bytes);
             }
 
-            public static float TargetSize
-            {
-                get
-                {
-                    return EditorPrefs.GetFloat(kComposerTargetSizeKey, 5f);
-                }
-
-                set
-                {
-                    if (TargetSize != value)
-                    {
-                        EditorPrefs.SetFloat(kComposerTargetSizeKey, value);
-                    }
-                }
-            }
+            static string Base64Encode(byte[] data) => Convert.ToBase64String(data);
+            static byte[] Base64Decode(string base64EncodedData) => Convert.FromBase64String(base64EncodedData);
         }
 
-        private static bool ShowCoreSettings
+        static void OnPostprocessAllAssets(
+            string[] importedAssets, string[] deletedAssets, string[] movedAssets, 
+            string[] movedFromAssetPaths, bool didDomainReload)
         {
-            get { return EditorPrefs.GetBool(kCoreSettingsFoldKey, false); }
-            set
-            {
-                if (value != ShowCoreSettings)
-                {
-                    EditorPrefs.SetBool(kCoreSettingsFoldKey, value);
-                }
-            }
+            if (didDomainReload)
+                EditorApplication.hierarchyWindowItemOnGUI += OnHierarchyGUI;
         }
```

Comment:
This is not called anywhere, do we need it?

Reply:
I thought so too, then I put a breakpoint.  It gets called magically by the infrastructure.